web/src/containers/BudgetSummary.js

```diff
@@ -43,8 +43,12 @@ DataRow.propTypes = {
 
 const DataRowGroup = ({ data, year }) => (
   <Fragment>
-    {categories.map(({ category, title }) => (
-      <DataRow category={category} data={data[category][year]} title={title} />
+    {categories.map(({ category, title }, i) => (
+      <DataRow
+        category={category}
+        data={data[category][year]}
+        key={i} // eslint-disable-line react/no-array-index-key
+        title={title} />
```

Comment:
> When you don’t have stable IDs for rendered items, you may use the item index as a key as a last resort.

https://reactjs.org/docs/lists-and-keys.html

If there's a better option than using array index as key in this instance, please let me know.

Reply:
I'd like to put more thought into this.  See the link here for some rationale as to why we shouldn't do this:  https://stackoverflow.com/questions/46735483/error-do-not-use-array-index-in-keys/46735689

Reply:
one thing I've done in the past is to use two variables I had handy, concatenated with a hyphen, like `${category}-${year}`

Reply:
> We don’t recommend using indexes for keys if the order of items may change. 

https://reactjs.org/docs/lists-and-keys.html

In this case, according to the react docs, I think it's safe to leave it as is. However, I'm happy to change the key values to whatever you would like them to be.

Reply:
Drilling down into the documentation for the `react/no-array-index-key` rule...

> If there is nothing unique about the items, [...] then you may want to disable this rule with an override.

https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/no-array-index-key.md#when-not-to-use-it

This code is iterating over a `const` defined earlier in the file: 

```
const categories = [
  { category: 'statePersonnel', title: 'State Staff' },
  { category: 'expenses', title: 'Other State Expenses' },
  { category: 'contractors', title: 'Private Contractor' },
  { category: 'combined', title: 'Subtotal' }
];
```

I think we're safe using the index as a key, in this case.

Reply:
agreed